_posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "Partner Highlight: Oracle Cloud Infrastructure Search Service with OpenSearch is GA!"
+authors:
+  - andhopp
```

Comment:
Let's do Andrew, David, and Vacha!

Reply:
Do you mean rearrange the ordering? I have all three in there.